bitrisescript/src/bitrisescript/bitrise.py

```diff
@@ -0,0 +1,218 @@
+import asyncio
+import logging
+import os
+from pprint import pformat
+from typing import Any
+
+from aiohttp_retry import RetryClient
+
+from scriptworker_client.exceptions import TaskVerificationError
+from scriptworker_client.utils import get_single_item_from_sequence
+
+BITRISE_API_URL = "https://api.bitrise.io/v0.1"
+
+log = logging.getLogger(__name__)
```

Comment:
suggestion: should we make the `log` object more context-aware? In case multiple jobs are running at the same time, it would provide us a way of differentiating each log line.

Not required for this initial PR, maybe something to just keep in mind?

Reply:
Good idea, I attempted to get this working via a logging adapter.. but the global state is shared across async contexts, so I think we'd need to pass the logging object everywhere. I think I'll just leave it for now..